app/assets/javascripts/modules/dashboard.js

```diff
@@ -52,23 +52,42 @@ angular.module('probedock.dashboard', [ 'probedock.api', 'probedock.orgs', 'prob
       controller: 'NewTestsLineChartCtrl',
       templateUrl: '/templates/new-tests-line-chart.html',
       scope: {
-        organization: '='
+        organization: '=',
+        projectId: '='
       }
     };
   })
 
   .controller('NewTestsLineChartCtrl', function(api, $scope) {
 
+    var chartConfig = {
+      written: {
+        url: '/metrics/newTestsByDay',
+        tooltipTemplate: '<%= value %> new tests on <%= label %>',
+        valueFieldName: 'testsCount'
+      },
+      run: {
+        url: '/metrics/reportsByDay',
+        tooltipTemplate: '<%= value %> runs on <%= label %>',
+        valueFieldName: 'runsCount'
+      }
+    };
+
     $scope.chart = {
       data: [],
       labels: [],
       params: {
         projectIds: [],
-        userIds: []
+        userIds: [],
+        type: 'written'
       },
       options: {
         pointHitDetectionRadius: 5,
-        tooltipTemplate: '<%= value %> new tests on <%= label %>'
+        // Will be set from chartConfig based on chart type
+        tooltipTemplate: '',
+        scaleLabel: function(object) {
+          return '  ' + object.value;
```

Comment:
Please add a comment for this fix, if possible with the link to the source of the solution.